README.md

```diff
@@ -1 +1,34 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+## Portfolio-setup-and-mobile-version-skeleton
+
+## Built With
+
+- HTML
+- CSS
+- Lighthouse
+- Webhint
+- Stylelint
+
+## Getting Started
+
+#### cloning project
+git clone git@github.com:collinsmezie/Portfolio-setup-and-mobile-version-skeleton.git
+
+## Getting packages and debuging with Stylelint npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
```

Comment:
- [ ] Lets make readme clear and professional. In this case it is not clear where is title of the section and why you suddenly writing about installing linters in heading 🙈